roles/openshift_openstack/defaults/main.yml

```diff
@@ -105,3 +105,179 @@ openshift_openstack_provision_user_commands: []
 # cloud-config
 openshift_openstack_disable_root: true
 openshift_openstack_user: openshift
+
+# security groups
+openshift_openstack_cns_secgroup_rules:
```

Comment:
It's a small thing but could we keep the same order as the *-secgrp resources have in the Heat template?

So that would be: common, pod access, master, etcd, node, infra, cns, lb. I like that order a little better (not a big deal though!)

Reply:
Sure, I'll update that!